### PR TITLE
Add libseccomp, bubblewrap and xdg-dbus-proxy as build dependencies ...

### PR DESCRIPTION
**Pull request by psaavedra**
... if ENABLE_BUBBLEWRAP_SANDBOX

### CODE REVIEW

File: recipes-flatpak/xdg-dbus-proxy/xdg-dbus-proxy_0.1.1.bb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+HOMEPAGE = "https://github.com/flatpak/xdg-dbus-proxy"
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://COPYING;md5=4fbd65380cdd255951079008b364516c"
+
+SRC_URI = " \
+    git://git@github.com/flatpak/xdg-dbus-proxy;protocol=https \
+"
+SRCREV = "7ebcc6856c6a1ee9571fa19c45999e86d44d3e5b"
```
Comment (philn) [inline recipes-flatpak/xdg-dbus-proxy/xdg-dbus-proxy_0.1.1.bb]: Why using a git repo for a release?

File: recipes-flatpak/bubblewrap/bubblewrap_git.bb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+LIC_FILES_CHKSUM = "file://COPYING;md5=5f30f0716dfdd0d91eb439ebec522ec2"
+
+DEPENDS = "libcap"
+
+SRC_URI = " \
+    git://git@github.com/projectatomic/bubblewrap;protocol=https \
+"
+SRCREV = "3ebe54af1b2acb702381ec9b5610b60758d3d163"
+
+PV = "2016.8+git${SRCPV}"
```
Comment (philn) [inline recipes-flatpak/bubblewrap/bubblewrap_git.bb]: Why is a git snapshot needed? Perhaps the latest release would be enough?

File: recipes-flatpak/xdg-dbus-proxy/xdg-dbus-proxy_0.1.1.bb (hunk 1 above)
Comment (psaavedra) [reply recipes-flatpak/xdg-dbus-proxy/xdg-dbus-proxy_0.1.1.bb]: Fixed in the revision

File: recipes-flatpak/bubblewrap/bubblewrap_git.bb (hunk 2 above)
Comment (psaavedra) [reply recipes-flatpak/bubblewrap/bubblewrap_git.bb]: Fixed in the revision

**Pull request merged**